erizo_controller/erizoClient/src/utils/SdpHelpers.js

```diff
@@ -62,4 +62,15 @@ SdpHelpers.setParamForCodecs = (sdpInfo, mediaType, paramName, value) => {
   });
 };
 
+SdpHelpers.filterAbsSendTimeAudio = (sdp) => {
```

Comment:
why don't you just remove it from the rtp_media_config or use a specific codec configuration without it?

Reply:
if I remove it from the rtp_media_config I will remove it for every browser. this is more specific to Edge. Am I wrong?